### PR TITLE
New Raspberry Pi Distributions Cheat Sheet

### PR DESCRIPTION
**Pull request by end222**
## Description of new Instant Answer, or changes
It gives a list of the distributions available for the Raspberry Pi and gives links the their respective webpages

## Testing & Review
To be completed by Community Leader (or DDG Staff) when reviewing Pull Request

**Pull Request**
- [x] Title follows correct format (Specifies Instant Answer + Purpose)
- [x] Description contains a valid Instant Answer Page Link (e.g. https://duck.co/ia/view/my_ia)

**Instant Answer Page** (for new Instant Answers)
- [x] Instant Answer page is correctly filled out and contains:
    - The **Title** is appropriately named and formatted
    - The **IA topics** are present and appropriate
    - The **Description** is clear and coherent
    - **Source Name** exists if applicable
    - All **Example Queries** trigger on [Beta](https://beta.duckduckgo.com/)

**Code**
- [x] Adheres to the [DuckDuckGo Style Guide](https://docs.duckduckhack.com/resources/code-style-guide.html)
- [x] Behaviour is appropriately tested. If improvement, tests are adequately extended.
- [x] There is no unnecessary files in place (such as editor config files)
- [x] There is no API keys / secrets present
- [x] Tests are passing (run `$ duckpan test <goodie_id>`)
    - Tester should report any failures

**Ready to merge?**

- [x] Has this IA been deployed to and tested on [beta.duckduckgo.com](https://beta.duckduckgo.com/)?
- [ ] For larger commits, has this been approved be more than one community member?
- [ ] The number of reviews is appropriate for this type of PR
- [ ] The commit message is clear, coherent and fitting

**Pull Request Review Guidelines**: https://docs.duckduckhack.com/programming-mission/pr-review.html

---

Instant Answer Page: https://duck.co/ia/view/raspberry_pi_distributions_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/raspberry-pi-distributions.json [hunk 1]
```diff
@@ -0,0 +1,70 @@
+{
+    "id": "raspberry_pi_distributions_cheat_sheet",
+    "name": "RPiDistributions",
```
Comment (adityatandon007) [inline share/goodie/cheat_sheets/json/raspberry-pi-distributions.json]: give some spaces in the `name`  value

File: share/goodie/cheat_sheets/json/raspberry-pi-distributions.json [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    "description": "List of available distributions for the Raspberry Pi",
+
+    "metadata": {
+        "sourceName": "elinux",
+        "sourceUrl" : "http://elinux.org/RPi_Distributions"
+    },
+
+    "aliases": [
+        "rpi distros", "raspberry pi distributions", "raspberry pi distros"
+    ],
```
Comment (adityatandon007) [inline share/goodie/cheat_sheets/json/raspberry-pi-distributions.json]: you can add some more aliases like `rasp pi distros` and `raspberry distros`

File: share/goodie/cheat_sheets/json/raspberry-pi-distributions.json [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+                "key": "Arch Linux ARM"
+            },
+            {
+                "link": "https://wiki.gentoo.org/wiki/Project:ARM",
+                "key": "Gentoo Linux"
+            }
+        ],
+        "Network Centered": [
+            {
+                "link": "https://www.kali.org",
```
Comment (adityatandon007) [inline share/goodie/cheat_sheets/json/raspberry-pi-distributions.json]: change this link for the `kali` it points to the kali linux -- a linux distro not a rpi distro
change this with this `http://docs.kali.org/kali-on-arm/install-kali-linux-arm-raspberry-pi`

**Comment by adityatandon007** [comment]
@end222 you can also add some more distro if you want

**Comment by end222** [comment]
@adityatandon007 Okay, I will fix the kali link and add some more distros as soon as I have time

**Comment by mintsoft** [comment]
Many thanks @end222 Apologies for the epic delay getting around to this

**Comment by daxtheduck** [comment]
@end222 this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Pull request merged**